### DISCUSSION TITLE
Use?

### DISCUSSION THREAD

**Discussion by pnes** [marioparaschiv/nitro-sniper#89] [Q&A] [answered]
I know this sounds dumb of me but... hoe exactly do I use it it loaded and says logged in but then after that there's nothing. So if someone can tell me what to do I would be grateful

**Comment by MrD4rio** [comment, marked as answer]
The sniper will detect any nitro codes coming from the servers you're in and will try to redeem them. To check if it's working correctly you could try sending a random nitro code in a server where there's the sniping account(s) and see if it gets logged into the console.
